### DISCUSSION TITLE
join_geometry

### DISCUSSION THREAD

**Discussion by pterusgithub** [WannesMalfait/Blender-Add-ons#23] [Q&A] [answered]
Hello,

Could-it be possible to have an example on "how to use join_geometry() to connect several nodes" ?
Thank you

**Comment by WannesMalfait** [comment, marked as answer]
Hello, sadly I have not added support for multi-input sockets to the add-on yet. At the moment it treats "multi-input" sockets as if they were just a normal socket. 

This needs to be implemented at some point: #24
